tests/scenarios/breastCancerTestScenarios.yaml

```diff
@@ -17,4 +17,4 @@ scenarios:
 
 reporting:
 - module: junit-xml
-  filename: ../results/sampleTest.xml
\ No newline at end of file
+  filename: /bzt-configs/tests/results/sampleTest.xml
```

Comment:
non-blocking request

I'm going to assume this works based on the example in the [README](https://github.com/Alvearie/quality-measure-and-cohort-service/blame/7961c2f5c76e45f10d58bd83f3c8e84369455095/tests/README.md#L37),
but I feel like it should at least be better documented?

Reply:
Good point, this change was made so the result xml file would be put in the shared drive in the docker container. Jai, want to add some details in the README about some of these settings?